dev-requirements.txt

```diff
@@ -28,4 +28,4 @@ wrapt==1.10.8
 scikit-learn==0.20.3
 pymongo==3.8.0
 py-cpuinfo==4.0
-
+pre-commit==1.18.0
```

Comment:
should we add black here? when does it get downloaded?

Reply:
I first had it there, but then the py35 build fails, because it installs the `dev-requirements` but black needs py36 to run.

Reply:
so `black` will get downloaded when you commit the first time after running `pre-commit install`. It is installed in its own virtual environment that does not contain anything else.

Reply:
I see, that makes sense. I'll merge it then.